Makefile

```diff
@@ -31,6 +31,18 @@ dev_build:
 dev_logs:
 	docker-compose -f docker-compose.dev.yml logs -f
 
+dev_up:
+	echo $@
+	docker-compose -f docker-compose.dev.yml up
+
+dev_down:
+	echo $@
+	docker-compose -f docker-compose.dev.yml down
+
+dev_build:
+	echo @
+	docker-compose -f docker-compose.dev.yml build
```

Comment:
These changes should possibly be in a separate PR (if they're not already) since they're not related to the Groups component.

cc @wittejm

Reply:
That's a git behavior I don't really get, because the changes from mine and Matt's commits were already merged into Master. @darenschaad, maybe you could remove those commits from your branch using `git rebase -i <base-branch>` or some other git-fu?

Reply:
I have had this issue in the past and it had to do with some things being merged into my local master branch that were then for some reason seen as additions to the remote master when I tried to merge the branch back into the remote eventually.

If you are still having issues with this tomorrow I'll be at the meetup and can help puzzle it out!

Reply:
This can occur when you merge in changes from another branch. This should be able to be fixed with a rebase. I’d be happy to walk anyone through the process. There are also a couple videos I posted in the #random slack channel that shows how to rebase.

Reply:
To avoid this in the future I’d advise to rebase on top of master instead of pulling from master. If interested I can walk you through a workflow that I find helpful.